packages/rpcview.vm/rpcview.vm.nuspec

```
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <id>rpcview.vm</id>
+    <version>0.3.1.20231018</version>
+    <authors>silverf0x</authors>
+    <description>RpcView is an open-source tool to explore and decompile all RPC functionalities present on a Microsoft system</description>
+    <dependencies>
+      <dependency id="common.vm" />
+      <dependency id="vcredist140" version="14.36.32532"/>
```

```suggestion
      <dependency id="vcredist140" version="[14.36.32532]" />
```

We should ping the dependency to ensure we test updates or limit it to a lower range like `version="[14.36.32532, 14.37)"`. This dependency is used also in [nmap](https://github.com/mandiant/VM-Packages/blob/a21b189ce92c24c1ca8a85e5d5692fdfccca0cb0/packages/nmap.vm/nmap.vm.nuspec#L12). I think we should create a metapackage to ensure both use the same version and avoid conflicts. I have added this to https://github.com/mandiant/VM-Packages/issues/673 (we can address it in a different PR).